### ISSUE TITLE
Request: Video settings

### ISSUE THREAD

**Issue by Rainer321** [BigRedBot/NeverwinterInvokeBot#129]
Hey BigRedBot,

First of all, thanks for your great work! It is really appreciated and saves me a lot of time doing the boring stuff every day which makes me free to do the fun stuff :-).

What I'm noticing is that my visuals are slightly off compared to the images that are provided with the bot. This means that I have to adapt all the images. This was fine for the invocation part, but in the professions bot part I've found a more difficult issue to solve myself. I've noticed that an image that should be found multiple times (like "collect reward") is not found everywhere - and I've tried several times to update the image. It seems that they look slightly different on every location on the screen. This might be because of some video settings. I've already played around with the resolution settings but not yet with results (I've noticed that the bot rescales the neverwinter screen to 1024x768 so I've tried that resolution - native is 1920x1080).

Could you provide us with all your visual settings? So resolution, AA, aspect ratio, resolution scale etc etc. Maybe in the form of images from your settings..? I would love to just be able to use the default images that are provided with the bot.

Thanks in advance!

Kind regards,
Rainer

**Comment by BigRedBot** [comment]
Make sure the resolution scale is set to 1.0. That will mess it up for sure if it is not set to that.

**Comment by Rainer321** [comment]
I've doublechecked the resolution scale and that is nicely set to 1.0.

**Comment by Rainer321** [comment]
You probably forgot about this (totally understandable). So here is a kind reminder ;-)

**Comment by Usaal** [comment]
You still can't get it to work on your display?.. have you set the game to default, heres what works for me sorry i use dual screens, so my screen shot is alittle miss leading.. but if you look at how the back ground is stretched/skewed youll see the resolutions are different so youll know where the game screen is located

I use 2 displays, the second being a 42 ince sharp tv set, off to my left, its resolution is set at 1280x720, My pc display is a 17.5 in x 8.3 in x 14.3 in.. and it is set at 1440x900

Here is me fishing
![mydisplayworksjust fine here i am fishing](https://cloud.githubusercontent.com/assets/24265790/24382325/759b328e-1323-11e7-869a-a6a60d6a1477.png)

and here is me displaying my in game display settings, the bot does sometimes have to adjust my windowed game for me due to me grabbing a edge from time to time, but its rare, and the bot does it for me

![myingamedisplaysettings](https://cloud.githubusercontent.com/assets/24265790/24382354/b2a90df4-1323-11e7-99ee-1755f019045a.png)

disregard if this has no relevance to your situation

- Usaal

**Comment by BigRedBot** [comment]
I am sorry for not getting back sooner to you. I did not forget. I left the email notification for this as a reminder in my inbox, so I am reminded of it every day. I do that for everything that I plan to get to. I have been repairing things in my house lately and did not get around to this yet.

If the information that the other (very kind) individual posted does not help you, let me know.

**Comment by Rainer321** [comment]
Hi guys,

Thanks for the help. I've tried to set my settings exactly like usaal, but to no avail. After some looking into the options I noticed that there also are Interface settings like "Interface scale" which probably is a big influence. I've reset the interface settings to default (which amonst others change the interface scale from 0.9 to 1.0 and voila! Everything works now as intended with the original settings...

![The problem value](https://cloud.githubusercontent.com/assets/24937219/24420052/bb00baaa-13f1-11e7-925e-b15ce59e4dd5.png)

If anyone else has problems that the images are not working - this is probably the fix (other than resolution scale)
